### DISCUSSION TITLE
Dịch restricted Boltzmann machine, deep belief network

### DISCUSSION THREAD

**Discussion by rootonchair** [mlbvn/handson-ml2-vn#26] [Terminology] [answered]
Em chưa có đề xuất nào cho từ này và hiện đang giữ nguyên. Có một cách dịch là `mạng tin sâu` nhưng `tin` ở đây cũng có nghĩa là `thông tin` nên em thấy chưa ổn lắm 

Đề xuất hiện tại:
* `restricted Boltzmann machine` -> `máy Boltzmann giới hạn`
* `deep belief network` -> `mạng niềm tin sâu`

@mlbvn/handson-ml2

**Comment by minhduc0711** [comment]
mình thấy wiki dịch là `mạng niềm tin sâu`, nghe được không nhỉ?

**Reply by rootonchair** [reply]
Em thấy cũng ổn ạ.
Source cho anh em: https://vi.wikipedia.org/wiki/H%E1%BB%8Dc_s%C3%A2u#M%E1%BA%A1ng_ni%E1%BB%81m_tin_s%C3%A2u_(Deep_belief_network)

**Comment by minhduc0711** [comment]
ping thread, mọi người góp ý giúp 2 từ này nhé

**Comment by minhduc0711** [comment, marked as answer]
hmm chắc tạm chốt vậy
* `restricted Boltzmann machine` -> `máy Boltzmann giới hạn`
* `deep belief network` -> `mạng niềm tin sâu`
